tests/test_scylla_repository.py

```diff
@@ -111,14 +127,14 @@ def test_setup_release_enterprise_no_unified_package(self, version, expected_cdi
         assert packages.scylla_tools_package
         assert packages.scylla_jmx_package
 
-    def test_setup_unstable_master_new_url(self):
-        cdir, packages = scylla_setup(version="unstable/master:2021-01-18T15:48:13Z", verbose=True, skip_downloads=True)
-        assert '2021-01-18T15_48_13Z' in cdir
-        assert packages.scylla_unified_package is None
-        assert packages.scylla_package == 'https://s3.amazonaws.com/downloads.scylladb.com/unstable/scylla/master/relocatable/2021-01-18T15:48:13Z/scylla-package.tar.gz'
-        assert packages.scylla_tools_package == 'https://s3.amazonaws.com/downloads.scylladb.com/unstable/scylla/master/relocatable/2021-01-18T15:48:13Z/scylla-tools-package.tar.gz'
-        assert packages.scylla_jmx_package == 'https://s3.amazonaws.com/downloads.scylladb.com/unstable/scylla/master/relocatable/2021-01-18T15:48:13Z/scylla-jmx-package.tar.gz'
-
+    @pytest.mark.parametrize(argnames=['version', 'expected_cdir', 'scylla_debug'], argvalues=[
+        ("unstable/master:2021-01-18T15:48:13Z", "2021-01-18T15:48:13Z", False),
+        ("unstable/master:2021-01-18T15:48:13Z:debug", "2021-01-18T15:48:13Z", True),
+    ])
```

Comment:
Can we test this also with enterprise url please?

Reply:
https://github.com/scylladb/scylla-ccm/pull/598/commits/aa386e60413c4ca2b93b03655801b44bb4cd7715